WhaleSpotting.UnitTests/Controllers/SightingControllerTests.cs

```diff
@@ -83,8 +83,7 @@ public async void CreateSighting_CalledWithNewSighting_ReturnsCreatedResult()
                 Description = "was nice",
                 OrcaType = "",
                 OrcaPod = "",
-                UserId = currentUser.Id,
-                Username = currentUser.NormalizedEmail,
+                Username = "FakeUser",
```

Comment:
this should be currentUser.UserName, and you should set the UserName not NormalizedEmail on the currentUser

Reply:
Actually can you go through and change all the locations where we are setting currentUsers NormalizedEmail to set the UserName instead